pkgs/applications/science/machine-learning/streamlit/default.nix

```
@@ -0,0 +1,36 @@
+{   lib, buildPythonApplication, fetchPypi
+  , altair, astor, base58, blinker, boto3, botocore, click, enum-compat
+  , future, pillow, protobuf, requests, toml, tornado, tzlocal, validators, watchdog
+  , jinja2, setuptools
+}:
+
+buildPythonApplication rec {
+  pname = "streamlit";
+  version = "0.47.4";
```

i would move format up, sorry for the last minute changes
```suggestion
  version = "0.47.4";
  format = "wheel";
```